docs/crowdin.yaml

```diff
@@ -0,0 +1,7 @@
+project_identifier_env: CROWDIN_PROJECT
+api_key_env: CROWDIN_API_KEY
+base_path: .
+
+files:
+  - source: '/locale/docs/en/LC_MESSAGES/*.po'
```

Comment:
Paths are wrong, should point to your local directories, you could adjust the `base_path` above